util/os/osx.zig

```diff
@@ -0,0 +1,31 @@
+const std = @import("std");
+
+pub fn getCpuName(allocator: std.mem.Allocator) ![]const u8 {
+    return try exec(allocator, &.{ "sysctl", "-n", "machdep.cpu.brand_string" });
+}
+
+pub fn getCpuCores(allocator: std.mem.Allocator) !u32 {
+    const coresString = try exec(allocator, &.{ "sysctl", "-n", "hw.physicalcpu" });
+    defer allocator.free(coresString);
+
+    return std.fmt.parseInt(u32, coresString, 10) catch |err| {
+        std.debug.print("Error parsing CPU cores count: {}\n", .{err});
```

Comment:
Better use scoped logging here so that the source of the information is clear (applies to all std.debug.print calls in osx.zig, windows.zig and linux.zig)

Reply:
Good catch 👍